recipes/u-boot/u-boot-tools.inc

```diff
@@ -13,35 +13,22 @@ S = "${SRCDIR}/u-boot-${PV}"
 MIRRORS =. """http://ftp.denx.de/pub/u-boot/ ftp://ftp.denx.de/pub/u-boot/
 """
 
-DEPENDS:sdk = "native:u-boot-tools-bmp-logo"
+EXTRA_OEMAKE = 'HOSTCC="${CC}" \
+    HOSTLDFLAGS="${HOST_LDFLAGS}"'
 
-DO_COMPILE_FIX_SDK_BUILD = ""
-DO_COMPILE_FIX_SDK_BUILD:sdk = "do_compile_fix_sdk_build"
-do_compile[prefuncs] += "${DO_COMPILE_FIX_SDK_BUILD}"
-
-do_compile_fix_sdk_build() {
-    sed 's#$(obj)./bmp_logo $(LOGO_BMP) >$@#${BUILD_SYSROOT}/bin/bmp_logo $(LOGO_BMP) >$@#' -i ${S}/tools/Makefile
-}
-
-PARALLEL_MAKE = ""
-EXTRA_OEMAKE = 'HOSTCC="${CC}" HOSTSTRIP=":" \
-    HOSTLDFLAGS="${HOST_LDFLAGS}" \
+EXTRA_OEMAKE:machine = ' \
+    LDFLAGS="${LDFLAGS}" \
     CROSS_COMPILE="${HOST_PREFIX}"'
 
 RECIPE_FLAGS = "u_boot_tools_config"
-DEFAULT_USE_u_boot_tools_config = "eNET_config"
+DEFAULT_USE_u_boot_tools_config = "defconfig"
 
 do_configure () {
     oe_runmake ${USE_u_boot_tools_config}
 }
 
-MAKE_TARGETS = "tools-all"
-
-do_compile[prefuncs] += "do_compile_fix_gdb_include_path"
-
-do_compile_fix_gdb_include_path() {
-    sed 's#\(BFD_ROOT_DIR =\).*#\1 ${HOST_SYSROOT}${includedir}#' -i ${S}/config.mk
-}
```

Comment:
if something no longer applies, you are expected to try to figure out if it is still needed. Otherwise you push all that to me instead :) if the commit introducing it does not explain it, and you can't figure out how to make it work again then we might decide to drop it. But then you add that explanation in your commit message, so it becomes clear to the guy running into the original issue when using your updated recipe, why it was dropped.

In this case git blame reveals "u-boot/u-boot-tools: Fix compile on newer 32-bit distros" so, would probably be nice to test if that is still a problem. If you can't reproduce it, we drop it with a note in the commit msg